content/docs/user-guide/pipelines/index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Pipelines Management in DVC
+---
+
+# What is a (DVC) pipeline?
+
+A machine learning pipeline is like an assembly line in a factory. As raw
```

Comment:
I like the analogy but it's a bit general i.e. any pipeline is like an assembly line. What makes an ML pipeline special?

Reply:
"Pipeline" in software is usually shell pipes, like `ls -t | head -n 3`. "Assembly line" analogy is to counter that actually. I think if we start from the general pipelines, we may need to touch the shell pipes to avoid confusion. This was just to say "pipelines we talk in ML are not pipelines in shell."

Reply:
If we call the document **ML Pipelines** then we don't have to disambiguate here 🙂